xbmc/FileItem.cpp

```diff
@@ -1689,6 +1691,22 @@ void CFileItem::SetFromMusicInfoTag(const MUSIC_INFO::CMusicInfoTag &music)
     m_strPath = music.GetURL();
   m_bIsFolder = URIUtils::HasSlashAtEnd(m_strPath);
 
+  const std::string thumb = music.GetThumbURL();
```

Comment:
Out of curiosity, can't this be made another entry on the art map (or even a property) instead of a new member variable on fileitem? Looks like some effort was made in the past to get rid of art specific variables on the fileitem class (moving all them to art) and with this change a new one is being reintroduced.

Reply:
property seems good. Thanks for the hint.